src/assignment/hooks/useGetAssignmentsEditStatuses.tsx

```diff
@@ -4,15 +4,13 @@ import { AssignmentService } from '../assignment.service';
 
 export const useGetAssignmentsEditStatuses = (
 	assignmentsIds: string[],
-	enabled: boolean,
-	interval: number,
 	options: {
 		enabled: boolean;
-		refetchInterval: number;
-		refetchIntervalInBackground: boolean;
+		refetchInterval: number | false;
+		refetchIntervalInBackground?: boolean;
 	} = {
-		enabled: enabled || false,
-		refetchInterval: interval || 0,
+		enabled: false,
+		refetchInterval: false,
 		refetchIntervalInBackground: true,
 	}
```

Comment:
refetchIntervalInBackground is also false by default, my suggestion was not complete. sorry
```
	options: {
		enabled: boolean;
		refetchInterval: number | false;
		refetchIntervalInBackground?: boolean;
	} = {
		enabled: true,
		refetchInterval: false,
		refetchIntervalInBackground: false,
	}
```